backend/src/apiserver/resource/model_converter.go

```diff
@@ -199,15 +199,17 @@ func (r *ResourceManager) toModelResourceReferences(
 		if err != nil {
 			return nil, util.Wrap(err, "Failed to find the referred resource")
 		}
-		modelRef := &model.ResourceReference{
-			ResourceUUID:  resourceId,
-			ResourceType:  resourceType,
-			ReferenceUUID: apiRef.Key.Id,
-			ReferenceName: referenceName,
-			ReferenceType: modelReferenceType,
-			Relationship:  modelRelationship,
+		if apiRef.Key.Type != api.ResourceType_NAMESPACE {
```

Comment:
Just curious, why is namespace skipped here?

Reply:
This function is called when the backend tries to construct the data structure for the db. Currently, we haven't intended to add the namespace field in the db yet. We will in the near future.